src/services/ledger/index.ts

```diff
@@ -104,6 +104,41 @@ export const LedgerService = (): ILedgerService => {
     }
   }
 
+  const txVolumeSince = async ({
+    liabilitiesAccountId,
+    timestamp,
+  }: {
+    liabilitiesAccountId: LiabilitiesAccountId
+    timestamp: UnixTimeMs
```

Comment:
we are using Date and unixtime, maybe we should unify how we handle dates/times cc @samerbuna @bodymindarts

Reply:
I don't mind having multiple different representations.
There are always 3 layers of data representation
- how is it persisted?
- what is the domain type?
- what does the external caller want?

What layer are you suggesting to unify here?

Reply:
I was just looking it over again and I think in this specific instance the suggestion makes sense since we convert the unix timestamp [back to a `Date` object](https://github.com/GaloyMoney/galoy/blob/4b5353eb09f353ed0dc6df5d4fb9bf46541a1a79/src/services/ledger/index.ts#L121) when passing it to the Mongoose method.

Done here: https://github.com/GaloyMoney/galoy/pull/583/commits/63bfa7a9617ee483ab34fb2d2589a1ca18e56cde